theme_customizations/partials/footer.html

```diff
@@ -0,0 +1,99 @@
+  FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+  IN THE SOFTWARE.
+-->
+
+{%- import "partials/language.html" as lang with context -%}
+
+<!-- Application footer -->
+<footer class="md-footer">
+
+  <!-- Link to previous and/or next page -->
+  {%- if page.previous_page or page.next_page -%}
+    <div class="md-footer-nav">
+      <nav class="md-footer-nav__inner md-grid">
+
+        <!-- Link to previous page -->
+        {%- if page.previous_page -%}
+          <a href="{{ page.previous_page.url | url }}"
+              title="{{ page.previous_page.title }}"
+              class="md-flex md-footer-nav__link md-footer-nav__link--prev"
+              rel="prev">
+            <div class="md-flex__cell md-flex__cell--shrink">
+              <i class="md-icon md-icon--arrow-back
+                    md-footer-nav__button"></i>
+            </div>
+            <div class="md-flex__cell md-flex__cell--stretch
+                  md-footer-nav__title">
+              <span class="md-flex__ellipsis">
+                <span class="md-footer-nav__direction">
+                  {{- lang.t("footer.previous") -}}
+                </span>
+                {{- page.previous_page.title -}}
+              </span>
+            </div>
+          </a>
+        {%- endif -%}
+
+        <!-- Link to next page -->
+        {%- if page.next_page -%}
+          <a href="{{ page.next_page.url | url }}"
+              title="{{ page.next_page.title }}"
+              class="md-flex md-footer-nav__link md-footer-nav__link--next"
+              rel="next">
+            <div class="md-flex__cell md-flex__cell--stretch
+                  md-footer-nav__title">
+              <span class="md-flex__ellipsis">
+                <span class="md-footer-nav__direction">
+                  {{- lang.t("footer.next") -}}
+                </span>
+                {{- page.next_page.title -}}
+              </span>
+            </div>
+            <div class="md-flex__cell md-flex__cell--shrink">
+              <i class="md-icon md-icon--arrow-forward
+                    md-footer-nav__button"></i>
+            </div>
+          </a>
+        {%- endif -%}
+      </nav>
+    </div>
+  {%- endif -%}
+
+  <!-- Further information -->
+  <div class="md-footer-meta md-typeset">
+    <div class="md-footer-meta__inner md-grid">
+
+      <!-- Copyright and theme information -->
+      <div class="md-footer-copyright">
+        {%- if config.copyright -%}
+          <div class="md-footer-copyright__highlight">
+            {{- config.copyright -}}
+          </div>
+        {%- endif -%}
+        See the source code and discussion page on
+        <a href="https://github.com/bids-standard/bids-specification">Github</a>
```

Comment:
I believe it should be `GitHub` not `Github`

Reply:
+1, addressed - thanks